test/external-e2e/manifest/testdriver.yaml

```diff
@@ -14,4 +14,4 @@ DriverInfo:
         controllerExpansion: true
```

Comment:
I see it failed as expected, pls set `controllerExpansion: false`, then test would pass with this PR.